<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDialog</class>
 <widget class="QDialog" name="OutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>633</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Face Recognition Attendance App</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <widget class="QTabWidget" name="holl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>831</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>Student Detector</string>
    </attribute>
    <widget class="QLabel" name="imgLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>371</width>
       <height>351</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>4</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;
background-color: #FFF;
border-radius: 25px; 
border: 2px solid #66CDAA;  </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="checkInButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>430</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;
background-color: #66CDAA;
border-radius: 25px;
</string>
     </property>
     <property name="text">
      <string>Check In</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="checkOutButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>510</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000;
background-color: #66CDAA;
border-radius: 25px;
</string>
     </property>
     <property name="text">
      <string>Check Out</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>421</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Details</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>124</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Status :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Clocked Time : </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="IDLabel">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NameLabel">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lvlLabel">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="StatusLabel">
         <property name="font">
          <font>
           <family>Roboto Condensed Light</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="HoursLabel">
           <property name="font">
            <font>
             <family>Roboto Condensed Light</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MinLabel">
           <property name="font">
            <font>
             <family>Roboto Condensed Light</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="Date_Label">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Time :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Time_Label">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Bstart">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>510</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff;
background-color: #0080ff;
border-radius: 25px;
</string>
     </property>
     <property name="text">
      <string>start camera</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Bstop">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>510</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff;
background-color: #990000;
border-radius: 25px;
</string>
     </property>
     <property name="text">
      <string>stop camera</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>450</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>450</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Multimedia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Compiler Design</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Expert Systems</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Algorithms</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="course">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>460</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Course:</string>
     </property>
    </widget>
    <widget class="QLabel" name="course_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>460</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Place:</string>
     </property>
    </widget>
    <widget class="QLabel" name="NameLabe">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QLabel" name="NameLabe2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>40</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Condensed Light</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>0</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Mode</string>
     </property>
     <widget class="QPushButton" name="darkk">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff;
background-color: black;
border: 2px solid #66CDAA;  
border-radius: 25px;
</string>
      </property>
      <property name="text">
       <string>dark</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="whitee">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
background-color: white;
border: 2px solid #66CDAA;  
border-radius: 25px;

</string>
      </property>
      <property name="text">
       <string>white</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Add Student</string>
    </attribute>
    <widget class="QLabel" name="id">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>National ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linefname">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #66CDAA;
border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="fname">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linelname">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>110</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #66CDAA;
border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="lname">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineid">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #66CDAA;
border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_28">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>660</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFF;
background-color: #66CDAA;
border-radius: 25px;
</string>
     </property>
     <property name="text">
      <string>Add Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_29">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>660</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFF;
background-color: #66CDAA;
border-radius: 25px;
</string>
     </property>
     <property name="text">
      <string>Delete Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_30">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>660</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFF;
background-color: #66CDAA;
border-radius: 25px;
</string>
     </property>
     <property name="text">
      <string>Edit Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="studentButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>430</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFF;
background-color: #4381B3;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string>Add Student</string>
     </property>
    </widget>
    <widget class="QLabel" name="level">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>270</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Level</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linelevel">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>270</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #66CDAA;
border-radius: 10px;
</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
